views/partials/search-form-fields.html

```diff
@@ -183,7 +183,7 @@
   <div class="controls">
     <button type="submit"><span>Search</span></button>
     {% if advancedSearchIsVisible %}
```

Comment:
It'll only appear if you already have some advanced search filters in place (e.g. http://explorer.opentrials.net/search?location=Romania)